lib/travis/build/script/rust.rb

```diff
@@ -58,6 +58,10 @@ def use_directory_cache?
           super || data.cache?(:cargo)
         end
 
+        def before_cache
+          sh.cmd 'rm -rf "$HOME/.cargo/registry/src"', timing: false, echo: false
```

Comment:
You may want a note that this is to remove the unpacked crate sources because there's no need to cache those.

Reply:
@kinnison Thanks. The documentation PR is found in https://github.com/travis-ci/docs-travis-ci-com/pull/2391. I appreciate your feedback there as well.